packages/trie/test/index.spec.ts

```diff
@@ -170,11 +170,7 @@ tape('testing deletion cases', function (tester) {
     trie: new Trie(),
     msg: 'without DB delete',
   }
-  const trieSetupWithDBDelete = {
-    trie: new Trie({ deleteFromDB: true }),
-    msg: 'with DB delete',
-  }
-  const trieSetups = [trieSetupWithoutDBDelete, trieSetupWithDBDelete]
+  const trieSetups = [trieSetupWithoutDBDelete]
```

Comment:
Since we only have one setup, I think we could now do without that array and the `for (const tripeSetup of trieSetups)` below